README.md

```diff
@@ -10,15 +10,37 @@ Sporkfish is a Python-based chess engine. Chess programming techniques, although
 
 ### Set up
 
-To quickly set up the environment, you can use docker. From a terminal in the root directory:
+As a general case `Using DevContainer with VSCode` or `Using Github Codespace`
 
-```
-docker pull kylchiu/sporkfish-dev:latest
-docker build -t kylchiu/sporkfish-dev:latest .
-docker run -it kylchiu/sporkfish-dev:latest
-```
+Install: 
+- [VSCode](https://code.visualstudio.com/download)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+#### Using DevContainer with VSCode 
+
+1. Make sure Docker Desktop is up and running.
+2. Open up the Command Palette and run `Dev Containers: Rebuild Container` 
+3. A warning might pop up saying `Not all host requirements in devcontainer.json are met by the Docker daemon.`, disregard this and continue. 
```

Comment:
Could we get rid of this if we get rid of of line 35-39 in the devcontainer.json file?